packages-native/bottom-sheet/src/BottomSheet.tsx

```diff
@@ -1,13 +1,13 @@
 import { createElement, Fragment, ReactElement } from "react";
-import { BottomSheetStyle, defaultBottomDrawerStyle } from "./ui/Styles";
-import { flattenStyles } from "@native-mobile-resources/util-widgets";
+import { BottomSheetStyle } from "./ui/Styles";
 import { CustomModalSheet } from "./components/CustomModalSheet";
 import { ExpandingDrawer } from "./components/ExpandingDrawer";
 import { NativeBottomSheet } from "./components/NativeBottomSheet";
 import { BottomSheetProps } from "../typings/BottomSheetProps";
+import { StyleSheet } from "react-native";
 
 export function BottomSheet(props: BottomSheetProps<BottomSheetStyle>): ReactElement {
-    const styles = flattenStyles(defaultBottomDrawerStyle, props.style);
+    const styles = StyleSheet.flatten(props.style);
```

Comment:
If I am not wrong we already receive just one object, so why flatten it?

Reply:
According to the typings we recieve an array.
Also i remember that we always recieve an array to not break the compatability.

https://github.com/mendix/widgets-resources/blob/6e0f2f7677d24ca8ba8d636e73a5eb43cfdad1cc/packages-native/bottom-sheet/typings/BottomSheetProps.d.ts#L29